.azure-pipelines/release-cli.yaml

```diff
@@ -381,7 +381,7 @@ stages:
                 Write-Verbose 'inlineNugetSignOperation = ''$(inlineNugetSignOperation)'''
                 $rid = '$(rid)'
                 $shouldSign = '$(sign)'
-                if ($shouldSign.ToLower() -eq 'true' -and '$(rid)' -eq 'nuget') {
+                if ($shouldSign.ToLower() -eq 'false' -and '$(rid)' -eq 'nuget') {
```

Comment:
```suggestion
                if ($shouldSign -eq $false -and '$(rid)' -eq 'nuget') {
```

Reply:
This won't do what I want it to do. Quoting the powershell page on booleans:

> The following types evaluate to $false:
>  - Empty strings like '' or ""
>  - Null values like $null
>  - Any numeric type with the value of 0

the string `false` doesn't evaluate to `$false`.

I think I'll update it to use `[bool]::Parse($shouldSign)` instead.